cluster/addons/addon-manager/Makefile

```diff
@@ -15,8 +15,8 @@
 IMAGE=staging-k8s.gcr.io/kube-addon-manager
 ARCH?=amd64
 TEMP_DIR:=$(shell mktemp -d)
-VERSION=v8.9
-KUBECTL_VERSION?=v1.11.3
+VERSION=v9.0
```

Comment:
bumped a major version because of the difference in the pruning whitelist that would get picked up from https://github.com/kubernetes/kubernetes/pull/72203/files#diff-32466b23d956bf8ce719bd73ff419872